docs/design-notes/protocols.md

```diff
@@ -166,6 +166,10 @@ but was recently dropped in Cassandra (version 4.0) and is likely to be
 dropped by Scylla in the future as well, so it is not recommended for new
 applications.
 
+By default, Scylla does not enable the Thrift server. In order to use it,
+it must be explicitly enabled by setting the `start_rpc` configuration option
+to true.
+
 By default scylla listens to the Thrift protocol on port 9160, which can be
```

Comment:
Maybe add a "When Thirft is enabled," in the beginning of this sentence, to clarify it doesn't contradict the new paragraph you added right before it.